storage-proofs/core/src/drgraph.rs

```diff
@@ -216,10 +217,14 @@ impl<H: Hasher> Graph<H> for BucketGraph<H> {
             "The number of metagraph nodes must be precisely castable to `f64`"
         );
 
+        let mut drg_seed = [0; 28];
+        let raw_seed = derive_porep_domain_seed("Filecoin_DRSample", porep_id);
+        drg_seed.copy_from_slice(&raw_seed[..28]);
```

Comment:
hmmmmm this seems very arbitrary, that we use the first 28 bytes

Reply:
I think you wrote the code using 28 bytes to seed `ChaChaRng`. If you think it should be changed, let me know how, please.